src/modules/esl-panel-group/core/esl-panel-group.ts

```diff
@@ -321,7 +322,11 @@ export class ESLPanelGroup extends ESLBaseElement {
     // activePanels = currentActivePanels - 1 (hide) + 1 if the event produced by 'before:show'
     const activeNumber = this.$activePanels.length - 1 + Number(selfHandled);
 
-    if (activeNumber < this.currentMinItems) return e.preventDefault();
+    if (activeNumber < this.currentMinItems) {
+      if (this.currentMinItems === 1 || this.currentMinItems === this.$panels.length) return e.preventDefault();
+      this.$panels.find(($panel: ESLPanel) => !$panel.open && $panel.show(this.mergeActionParams({event: e, target: $panel})));
```

Comment:
It's not a good idea to misuse the array methods.
It reduces readability a lot while it does not give any real benefits.